reporter/reporter.py

```diff
@@ -115,10 +121,81 @@ def notify():
     with CrateTranslator(DB_HOST, DB_PORT, DB_NAME) as trans:
         trans.insert([payload])
 
-    msg = "Notification successfully processed"
+    msg = 'Notification successfully processed'
     logger.info(msg)
     return msg
 
 
-if __name__ == '__main__':
-    app.run(host=LOCAL, port=8668)
+def query_1T1E1A(attrName, entityId, type=None):
+    """
+    See /entities/{entityId}/attrs/{attrName} in API Specification
+    quantumleap.yml
+    """
+    DB_HOST = os.environ.get('CRATE_HOST', 'crate')
+    DB_PORT = 4200
+    DB_NAME = "ngsi-tsdb"
```

Comment:
i would use a name that later on it can be easily used for the multi tenancy without the need of updating it.

e.g. default

Reply:
good idea

Reply:
One thing though, unless I misunderstood your idea, this change would break existing usages as of now (I think of the mexican apps, they'll need to rename their dbs). I believe it might be better to change to 'default' when implementing the multitenancy.

Reply:
1) let's make it configurable.
2) let's provide a guide on how to rename the database in case of upgrade

Reply:
since it's not related to API I'll do this later. Added reminder in #14